src/controller/FileBrowserController.vala

```diff
@@ -70,34 +70,34 @@ namespace BrickManager {
                             throw new IOError.PERMISSION_DENIED ("Cannot run files owned by system users.");
                         try {
                             var owner = file_info.get_attribute_string (FileAttribute.OWNER_USER);
+                            // openvt will unfortunately set the ownership of the new /dev/tty<num> to root:root
+                            // instead of root:tty. Even with root:tty the new application would fails to configure
+                            // the tty. We deal with that by changing the ownership of the new tty to belong to
+                            // the owner of the programm we launch.
+                            var ttyname_dir_template = "/tmp/brickman.XXXXXX".dup();
+                            var ttyname_dir = DirUtils.mkdtemp (ttyname_dir_template);
```

Comment:
I was hoping that this just created the name and didn't actually create a directory to make things easier. Guess I was wrong. But since you already did the work, this is OK. If we ever need any other temp files in brickman, we can refactor this directory out and use it elsewhere.